examples/ansible/kubeone/roles/kubeone_config/defaults/main.yml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019 The KubeOne Authors.
```

Comment:
As said on the meeting, I'd group required and recommended, and optional variables and add some comments to this file.

Example:

At the top have:
* Version
* Provider
* API endpoint
* Hosts
* Workers

And then like it is right now